spec/factories/event.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :event do
+    sequence(:title) { |n| "Event Title #{n}" }
+    sequence(:description) { |n| "Event Description #{n}" }
+    start_date Time.now
+    end_date { 7.days.from_now }
```

Comment:
Might update this to be (rand(60)+1).days.from_now instead to give it a bit more range to cover more cases for business logic, but don't think it's necessary for the specs as they are now